README.md

```diff
@@ -21,61 +23,67 @@ The CNCF TOC is the technical governing body of the CNCF Foundation. It admits a
+
+The CNCF provides [list of services](https://www.cncf.io/services-for-projects/) for incubating and graduated projects hosted in the foundation. A subset of these services is also available for [Sandbox](https://www.cncf.io/sandbox-projects/) level projects.
+
+Do you want to contribute to a project? We have [outlined the process](https://github.com/cncf/toc/blob/master/process/README.md) to get started. If you have any questions or don't know where to start, please open an [issue](https://github.com/cncf/toc/issues).
 
-See the process for contributing a project [here](https://github.com/cncf/toc/blob/master/process/README.md). If you have any questions or don't know where to start, open an [issue](https://github.com/cncf/toc/issues).
 
 ## Meeting Time
 
-The TOC meets on the 1st Tuesday of every month at 8AM PT (USA Pacific).
+The TOC meets on the 1st Tuesday of every month at 8AM PT (USA Pacific Time; [Convert to local time zone](http://www.thetimezoneconverter.com/?t=8:00AM&tz=San%20Francisco)).  
 Project Presentations are on the 3rd Tuesday of every month.
 
 https://zoom.us/j/967220397
 
 Or Telephone:
 
-Dial: +1 646 558 8656 (US Toll) or +1 408 638 0968 (US Toll)
-
-+1 855 880 1246 (US Toll Free)
-+1 877 369 0926 (US Toll Free)
-
-Meeting ID: 263 858 603
+> Dial:  
+> +1 646 558 8656 (US Toll) or  
+> +1 408 638 0968 (US Toll)  
+> 
+> +1 855 880 1246 (US Toll Free)  
+> +1 877 369 0926 (US Toll Free)
+> 
+> International numbers available [here](https://zoom.us/zoomconference?m=ddKUsXGa2tGOHvCl4ccI0juqU7TZaCov).
+>
+> Meeting ID: 263 858 603
+>
+> NOTE: _Please use `*6` to mute/un-mute your phone during the call_
 
-International numbers available [here](https://zoom.us/zoomconference?m=ddKUsXGa2tGOHvCl4ccI0juqU7TZaCov)
+Here is a [Google calendar for all public CNCF events](https://goo.gl/eyutah).
 
-Here is a public Google calendar you can add the meetings to your calendar: https://goo.gl/eyutah
-
-NOTE: Please use *6 to mute/un-mute your phone during the call
-
-Link to a World Time Zone Converter [here](http://www.thetimezoneconverter.com/?t=8:00%20a.m.%20&tz=San%20Francisco&)
 
 ## Meeting Agenda and Minutes
 
-The meeting Working Doc is [here](https://docs.google.com/document/d/1jpoKT12jf2jTf-2EJSAl4iTdA7Aoj_uiI19qIaECNFc/edit#). This includes minutes from previous meetings.
+- [Meeting Working Doc](https://docs.google.com/document/d/1jpoKT12jf2jTf-2EJSAl4iTdA7Aoj_uiI19qIaECNFc/edit#). This includes minutes from previous meetings.
+- [Meeting agenda and presentations](docs/meeting_presentations.md).
+- [Archive of community presentations](docs/scheduled_presentations.md)
 
-Meeting agenda and presentations are [here](docs/meeting_presentations.md).
+If you're interested in presenting at a TOC call about your project, please open a [GitHub issue](https://github.com/cncf/toc/issues) with the request. These requests are tracked in the [TOC project backlog](https://github.com/cncf/toc/projects/4).
```

Comment:
This is no longer accurate, SIGs are the project review spaces. Best to delete this entirely!

Reply:
Done!